torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -647,6 +647,9 @@ RegisterOperators reg({
           return [=](Stack& stack) {
             auto future = pop(stack).toFuture();
             if (future->completed()) {
+              if (future->hasError()) {
```

Comment:
Wouldn't this be better if we just made `Future::value()` do this? I.e. it resturns IValue on success but throws FutureError if there was an error.